### ISSUE TITLE
blocking accounts

### ISSUE THREAD

**Issue by shortwavesurfer2009** [dluvian/Nozzle#95]
I cannot see how to block an account that replied to my post as an example.

![Screenshot_20240303-105443_Nozzle](https://github.com/dluvian/Nozzle/assets/116814522/970fe886-d99b-4133-91d7-e09615536cc8)

This has absolutely nothing to do with my post, and I am so not interested. But my only options are follow, copy ID, and copy content as shown below.

![Screenshot_20240303-105746_Nozzle](https://github.com/dluvian/Nozzle/assets/116814522/29755b70-2395-4e0f-8e56-c6b47703d740)

**Comment by dluvian** [comment]
You are right. Blocking accounts is not implemented yet. But blocking is not the right approach in this case anyway. These types of scammers could just spin up new accounts for each reply. You blocking them would grow your block list without any effect.

I could add a "show hidden replies" button, under which replies from accounts with gray checkmarks are hidden.

**Comment by shortwavesurfer2009** [comment]
That's not a bad idea.
